src/openms/source/FORMAT/SqMassFile.cpp

```diff
@@ -45,22 +45,22 @@ namespace OpenMS
 
   void SqMassFile::load(const String& filename, MapType& map)
   {
-    OpenMS::Internal::MzMLSqliteHandler sql_mass(filename);
+    OpenMS::Internal::MzMLSqliteHandler sql_mass(filename, 0);
```

Comment:
0 = uninitialized id?

Reply:
yeah... somewhat.
The ID is only used for writing at the moment. Not for reading (because you'd need to peek the sqlite file to know that number).